mungegithub/mungegithub.go

```diff
@@ -83,17 +108,29 @@ func main() {
 		Use:   filepath.Base(os.Args[0]),
 		Short: "A program to add labels, check tests, and generally mess with outstanding PRs",
 		RunE: func(_ *cobra.Command, _ []string) error {
-			glog.Info(mungerutil.PrettyString(config))
+			optFlagsSpecified := config.Options.FlagsSpecified()
+			if config.path != "" && len(optFlagsSpecified) > 0 {
+				glog.Fatalf("Error: --config-map flag cannot be used with option flags. Option flag(s) %v were specified.", optFlagsSpecified)
```

Comment:
What you pass in --config-map is not an actual configMap object but one of its keys. `config-path` is also consistent with what is used in prow.